src/common/util/submit-data.test.js

```diff
@@ -3,30 +3,30 @@
  * @jest-environment-options {"html": "<html><head><script></script></head><body></body></html>", "url": "https://example.com/"}
  */
 
+import { faker } from '@faker-js/faker'
```

Comment:
As a principle for future reference, I'm curious about this choice. It seems like using faker instead of primitives makes test runs indeterministic and increases run time by a tiny fraction.

Reply:
Using fake data is better in unit tests to catch possible issues caused by hard coded values. It adds a small amount of overhead but it can help prevent nasty bugs later in production where a code path was mistakenly tied to a magic value unintentionally.